src/sdk/models/policies.ts

```diff
@@ -27,6 +33,8 @@ export interface PolicyItem extends TransferPlatformItem {
 	processed_on?: number;
 	root?: boolean;
 	type?: 'container' | 'item';
+	risk_level?: PolicyItemRiskLevel;
```

Comment:
`risk_level` will be within the tracking group

Reply:
The tracking group has a risk level associated with it?